syncnotification.go

```diff
@@ -2,10 +2,10 @@ package dcrlibwallet
 
 import (
 	"math"
+	"sync"
 	"time"
 
-	"github.com/decred/dcrwallet/chain"
-	"github.com/decred/dcrwallet/spv"
+	spv "github.com/decred/dcrwallet/spv/v3"
```

Comment:
Since we've renamed the `spv` module, can we use `github.com/raedahgroup/dcrlibwallet/spv` here? Then we can remove the replace directive in go.mod.